release-notes.md

```diff
@@ -0,0 +1,51 @@
+# v0.1.0
+
+This release has been tracked through the [0.1.0 project](https://github.com/intel/cloud-hypervisor/projects/1).
+
+Highlights for `cloud-hypervisor` version 0.1.0 include:
+
+### Shared filesystem
+
+We added support for the [virtio-fs](https://virtio-fs.gitlab.io/) shared file
+system, allowing for an efficient and reliable way of sharing a filesystem
+between the host and the `cloud-hypervisor` guest.
+
+See our [filesystem sharing](https://github.com/intel/cloud-hypervisor/blob/master/docs/fs.md)
+documentation for more details on how to use virtio-fs with `cloud-hypervisor`.
+
+### Inital direct device assignment support
+
+VFIO (Virtual Function I/O) is a kernel framework that exposes direct device
+access to userspace. `cloud-hypervisor` uses VFIO to directly assign host
+physical devices into its guest.
+
+See our [VFIO](https://github.com/intel/cloud-hypervisor/blob/master/docs/vfio.md)
+documentation for more detail on how to directly assign host devices to
+`cloud-hypervisor` guests.
+
+### Userspace IOAPIC
+
+`cloud-hypervisor` supports a so-called split IRQ chip implementation by
+implementing support for the [IOAPIC](https://wiki.osdev.org/IOAPIC).
+By moving part of the IRQ chip implementation from kernel space to user space,
+the IRQ chip emulation does not always run in a fully privileged mode.
+
+### Console over virtio
+
+`cloud-hypervisor` now exposes a `virtio-console` device to the guest. Although
+using this device as a guest console can potentially cut some early boot
+messages, it can reduce the guest boot time and provides a complete console
+implementation.
+
+The `virtio-console` device is enabled by default for the guest console.
+Switching back to the legacy serial port is done by selecting
+`--serial tty --console off` from the command line.
+
+### Unit testing
+
+We now run all unit tests from all our crates directly from our CI.
+
+### Integration tests parallelization
+
+The CI cycle run time has been significantly dropped by refactoring our
+integration tests implementation for running all of them in parallel.
```

Comment:
The CI cycle run time has been significantly reduced by refactoring our
￼ integration tests; allowing them to all be run in parallel.

Reply:
Thanks for the suggestion, fixed.